components/GymPlan/GymPlan.js

```diff
@@ -0,0 +1,13 @@
+import WorkoutSet from "./WorkoutSet";
+
+export default function GymPlan() {
+  return (
+    <>
+      <WorkoutSet setName="Warm-up" />
+      <WorkoutSet setName="Squat" />
+      <WorkoutSet setName="Bench" />
+      <WorkoutSet setName="Deadlift" />
+      <WorkoutSet setName="Assistant" />
+    </>
```

Comment:
```suggestion
    <form>
      <WorkoutSet setName="Warm-up" />
      <WorkoutSet setName="Squat" />
      <WorkoutSet setName="Bench" />
      <WorkoutSet setName="Deadlift" />
      <WorkoutSet setName="Assistant" />
    </form>
```
You should definitively add a form element here, so be semantically correct and to be able to work with the input values later :)